.gitlab-ci.yml

```diff
@@ -48,6 +50,20 @@ variables:
     - sh ./build.sh $GITLABCI_VERSION $BASE
     - docker push $BASE:$GITLABCI_VERSION
 
+.job_template: &release_template
+  only:
+    - master
```

Comment:
I'm not sure I understand this, but this should also happen on release branches, right?

Reply:
This is domjudge-packaging project, so it should only happen on the "accepted" CI file (as the other branches are mostly PRs)

Reply:
Ah, right. But then will there be a commit in `domjudge-packaging` that triggers this?

Reply:
Something like this:
https://github.com/mvr320/domjudge-scripts/blob/ac2a5bbc76aab6c056641b74aec9a75bb83c2031/DOMjudge_tag_release.sh#L41

We trigger from the server the packaging pipeline on gitlab. From there it will build the new docker containers and upload these to dockerHUB.

A simular pipeline can also be used to test the resulting tarball in more extensive scenarios (and in clean environments) or the debs etc.